gilda/ner.py

```diff
@@ -100,7 +99,8 @@ def get_brat(entities, entity_type="Entity", ix_offset=1, include_text=True):
     """
     brat = []
     ix_offset = max(1, ix_offset)
-    for idx, (raw_span, curie, start, end) in enumerate(entities, ix_offset):
+    for idx, (raw_span, match, start, end) in enumerate(entities, ix_offset):
+        curie = match.term.get_curie()
```

Comment:
The issue here is that the annotations we gather from the readers come already grounded and this function is used to create annotation for those sources as well.

Reply:
Oh, I see. One solution is to have this function handle it both ways, the other is to have a helper function that converts the more complex output from the newly proposed annotate() function to the for that get_brat() expects

Reply:
solution in 2d50223